_inc/client/at-a-glance/scan.jsx

```diff
@@ -106,55 +110,99 @@ class DashScan extends Component {
+						'jetpack'
+					),
+					{
+						keyLink: <a href={ this.props.siteAdminUrl + 'admin.php?page=vaultpress' } />,
+						supportLink: <a href={ getRedirectUrl( 'vaultpress-help' ) } />,
+					}
+				),
+			} );
+		}
+
+		// The VaultPress plugin is active and we received scanning data
+		const scanEnabled = get( vaultPressData, [ 'data', 'features', 'security' ], false );
+		if ( scanEnabled ) {
+			const threats = this.props.scanThreats;
+
+			// We found threats, so report them
+			if ( threats !== 0 ) {
+				return this.renderThreatsFound( threats, getRedirectUrl( 'vaultpress-dashboard' ) );
 			}
 
-			if ( scanEnabled ) {
-				// Check for threats
-				const threats = this.props.scanThreats;
-				if ( threats !== 0 ) {
-					return this.renderThreatsFound( threats, getRedirectUrl( 'vaultpress-dashboard' ) );
-				}
-
-				// All good
-				if ( vpData.code === 'success' ) {
-					return renderCard( {
-						status: 'is-working',
-						content: __( "No threats found, you're good to go!", 'jetpack' ),
-					} );
-				}
+			// No threats; all good
+			if ( vaultPressData.code === 'success' ) {
+				return renderCard( {
+					status: 'is-working',
+					content: __( "No threats found, you're good to go!", 'jetpack' ),
+				} );
 			}
 		}
 
-		if ( fetchingSiteData ) {
+		// At this point, either the plugin isn't active/installed, or we're not receiving scan data.
+		// We need to know this site's current plan for decision-making past this point.
+		if ( this.props.fetchingSiteData ) {
 			return renderCard( {
-				status: '',
 				content: __( 'Loading…', 'jetpack' ),
 			} );
 		}
 
-		const inactiveOrUninstalled = this.props.isVaultPressInstalled
-			? 'pro-inactive'
-			: 'pro-uninstalled';
-		const hasPremium = 'is-premium-plan' === planClass;
-		const hasBusiness = 'is-business-plan' === planClass;
+		const hasSitePlan = this.props.sitePlan !== false;
+		const scanningIncludedInPlan = [ 'is-premium-plan', 'is-business-plan' ].includes(
+			this.props.planClass
+		);
+
+		// If this site doesn't have scanning services as part of its plan,
+		// or if it has no plan, give the user a chance to upgrade
+		if ( ! hasSitePlan || ! scanningIncludedInPlan ) {
+			return renderCard( {
+				className: 'jp-dash-item__is-inactive',
+				status: 'no-pro-uninstalled-or-inactive',
+				overrideContent: this.getUpgradeBanner(),
+			} );
+		}
+
+		// VaultPress is installed, just not activated
+		if ( this.props.isVaultPressInstalled ) {
+			return renderCard( {
+				className: 'jp-dash-item__is-inactive',
+				status: 'pro-inactive',
+				content: [
+					<p className="jp-dash-item__description" key="inactive-scanning">
+						{ jetpackCreateInterpolateElement(
+							__(
+								'VaultPress is not active, <a>please activate</a> to enable automatic scanning for security for threats.',
```

Comment:
We do have an endpoint you could use here to help folks activate the plugin right from there. Here is an example with Akismet
https://github.com/Automattic/jetpack/blob/e20534fc3d97faa58db9252a030f2f5585b306be/_inc/client/at-a-glance/akismet.jsx#L62-L63

Reply:
Oh, that's slick! Added the same functionality to this PR. 👍